### ISSUE TITLE
Strange diff result

### ISSUE THREAD

**Issue by jradom** [marcdacz/compare-pdf#5]
I see the diff result where resolution is very low and it's not possible to understand what are the differences.  Also, the results under arrows are not correct.  Those sections are identical in both PDF files.  Please, advise

Thanks

jeff

![image](https://user-images.githubusercontent.com/20550461/98033458-5f09a900-1dca-11eb-9553-86926659e5b4.png)

**Comment by jradom** [comment]
I also tried comparing 2 images with some minor differences.  It looks like the text I had added has gray lines outside but not clear if it did highlight differences.  Is it a way to have a parameter that makes differences with red rectangles or at least highlighted with red color?
see screenshot below
![draftReport](https://user-images.githubusercontent.com/20550461/98153418-039df080-1e88-11eb-960b-b9244c152776.png)
![draftReport-expected](https://user-images.githubusercontent.com/20550461/98153450-0f89b280-1e88-11eb-90b7-5b2470bdf964.png)
![draftReport diff](https://user-images.githubusercontent.com/20550461/98153464-144e6680-1e88-11eb-8031-27acf3a430db.png)
